readme.txt

```diff
@@ -190,6 +190,24 @@ This version adds an option to refresh the malware scan results on demand, as we
 
 == Changelog ==
 
+= 1.8.22 =
+* Add "SSL existence check" to WordPress Security Recommendations
+* Add "Salt & Security Keys existence check" to WordPress Security Recommendations
+* Add "Salt & Security Keys age check" to WordPress Security Recommendations
+* Add "Admin account check" to WordPress Security Recommendations
+* Add "Single super-admin check" to WordPress Security Recommendations
+* Add "Use of 2FA plugin login recommendation" to WordPress Security Recommendations
+* Add "Use of backups plugin recommendation" to WordPress Security Recommendations
+* Add "Too much plugin check" to WordPress Security Recommendations
+* Add "File editing check" to WordPress Security Recommendations
+* Add "WordPress debug check" to WordPress Security Recommendations
+* Add "Basic hardening check" to WordPress Security Recommendations
+* Add a delete button on Last Logins sections
+* Add register of logs removal on Audit Logs
+* Fix display of Access File Integrity on NGINX/IIS servers
+* Update translation file to include new checks and buttons
```

Comment:
For the changelog, "Update translation file to include new checks and buttons" is really just a necessary part of all the other changes, rather than a change of its own. I suggest removing this line.